samples/HotAddSample/Startup.cs

```diff
@@ -12,11 +14,21 @@ namespace HotAddSample
     // will be reset before the end of the request.
     public class Startup
     {
+        public void ConfigureServices(IServiceCollection services)
```

Comment:
The WebListener instance is no longer added to ServerFeatures, so the sample was getting a NullReferenceException